base/gocb_utils.go

```diff
@@ -125,14 +118,12 @@ func getRootCAs(caCertPath string) (*x509.CertPool, error) {
 
 		ok := rootCAs.AppendCertsFromPEM(caCert)
 		if !ok {
-			return nil, errors.New("Invalid CA cert")
+			return nil, errors.New("invalid CA cert")
 		}
 
 		return rootCAs, nil
 	}
 
-	// We're purposefully ignoring the error here Due to the fact that the main error case is that this call is not
-	// supported in Windows.
-	rootCAs, _ := x509.SystemCertPool()
-	return rootCAs, nil
+	rootCAs, err := x509.SystemCertPool()
```

Comment:
Have we already had a discussion about how best to handle the windows error case?  Since the error is specifically "unable to load the system cert pool", I think logging that error and then returning an empty (non-nil) cert pool would be a better approach.  We'd expect this to fail to connect over couchbases unless insecureSkipVerify is set, but I don't think we need to pre-emptively error based on the failed SystemCertPool() call.